frontend/src/components/NavigateButton/NavigateButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { Button, Typography } from "@mui/material";
+
+import { useTranslation, Trans } from "react-i18next";
+
+import classNames from "classnames";
+
+import styles from "./styles.module.scss";
+
+type NavigateButtonProps = {
+  text: string;
+  isActiveSwitcher?: boolean;
+  isSwitcher?: boolean;
+  isMain?: boolean;
+  onClick?: () => void;
+};
+
+export const NavigateButton: React.FC<NavigateButtonProps> = ({
+  text,
+  isActiveSwitcher,
+  isSwitcher,
+  isMain,
+  onClick
+}) => {
+  return (
+    <Button
+      className={classNames({
+        [styles["activSwitcher"]]: isActiveSwitcher,
+        [styles["switcher"]]: isSwitcher,
+        [styles["default"]]: !isSwitcher && !isMain && !isActiveSwitcher,
```

Comment:
I think, that we can create separate variable for `!isSwitcher && !isMain && !isActiveSwitcher`

Something like this: 

```
const isDefault = !isSwitcher && !isMain && !isActiveSwitcher
        

[styles["default"]]: isDefault 


```